pkg/resource/aws/aws_instance.go

```diff
@@ -102,3 +102,10 @@ func (r *AwsInstance) TerraformType() string {
 func (r *AwsInstance) CtyValue() *cty.Value {
 	return r.CtyVal
 }
+
+func (r *AwsInstance) String() string {
+	if name, ok := r.Tags["Name"]; ok {
+		return name
```

Comment:
We should return the id in any case.

If a name exist : `i-abcdef12345678 (Name: my-instance)` if not only show `i-abcdef12345678`